### DISCUSSION TITLE
Link levels proportionally

### DISCUSSION THREAD

**Discussion by marobaro74** [xanderfrangos/twinkle-tray#675] [Ideas]
please add "link levels proportionally" feature.
Often two (or more) displays must be set to different brightness levels just to look the same. It would be very convenient to keep that relation intact when linking displays.

**Comment by xanderfrangos** [comment]
Hi @marobaro74,

The "Normalize Brightness" feature is meant to help with this exact concern. If you adjust the min/max levels so that all displays are roughly the same, the linked levels feature should keep the brightness consistent across all displays.

**Comment by marobaro74** [comment]
Hi Xander

Yes, you're right it does.
However It took me a while to get it working, probably because as you make
the necessary adjustments to separate monitors in "normalise brightness"
tab of the settings, those changes are not displayed in real time on
monitors, until you hit the Twinkle Tray icon. That refreshes the changes,
and then you see. But it is working blindly for a few seconds, then
refresh..
[image: image.png]

More intuitive would seem to be setting the separate screens interactively
in your nifty tray (with instant feedback), and then hit a button like
"link proportionally" (or normalized). you can use my icon idea if you like
:)
[image: image.png]

Hope you find it useful.
Many thanks,

Marek

**Comment by xanderfrangos** [comment]
Yeah, since the start I've been meaning to do a sort of "screen calibration" for the normalization feature, but never really got around to it.

**Comment by jeregon1** [comment]
Hey when I read about  proportionally linking what I'd like to proportionally link is the brightness and contrast of a display. I have the contrast with a max value of 50 (although hotkeys break that barrier) and would like that linking it with brightness would preserve the brightness/contrast relation of 2:1. It seems that the opposite is happening right now with the "Stop after this brightness level" slider, which if set to 50, then the relation is 1:2 instead of 2:1 breaking again the max contrast value (50).

Edit: I don't know why but now it actually does that after playing with the min and max values.
